cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -680,39 +678,6 @@ public void testFlows() throws Exception {
     Assert.assertEquals(200, deleteQueues(TEST_NAMESPACE1, WORDCOUNT_APP_NAME, WORDCOUNT_FLOW_NAME));
   }
 
-  @Test
```

Comment:
test removed because API was removed from v3/. A similar test exists for the v2/ version of this.